src/nrc/datamodel/management/commands/clean_old_notifications.py

```diff
@@ -0,0 +1,24 @@
+from datetime import timedelta
+
+from django.conf import settings
+from django.core.management.base import BaseCommand
+from django.db.models import F
+from django.utils import timezone
+
+from ...models import Notificatie
+
+
+class Command(BaseCommand):
+    def handle(self, **options):
+        date_limite = timezone.now() - timedelta(
+            days=settings.NOTIFICATION_NUMBER_OF_DAYS_RETAINED
+        )
+        notifications_filtrees = (
+            Notificatie.objects.annotate(
+                extracted_date=F("forwarded_msg__aanmaakdatum")
+            )
+            .filter(extracted_date__lt=date_limite)
```

Comment:
I'm not sure why do you need `annotate` here

```suggestion
            Notificatie.objects.filter(forwarded_msg__aanmaakdatum__lt=date_limite)
```